### PR TITLE
Interactive Visor

### PR DESCRIPTION
**Pull request by Ada18980**
Added a blindfold version of EmilyR's futuristic visor, complete with locking symbol and transparent/blind types. Has 4 levels:
- Transparent
- Light tint
- Heavy tint
- Blind

Tested chat messages, ability to modify while locked (the user intentionally has to have access to change the type, since it's all supposed to be software), and graphics of all 4 types (including lock symbol)

Also modified the original visor to have the same buygroup

### CODE REVIEW

File: BondageClub/Assets/Female3DCG/Female3DCG.js [hunk 1]
```diff
@@ -2848,6 +2848,11 @@ var AssetFemale3DCG = [
 		Asset: [
 			{ Name: "ClothBlindfold", Value: 15, Time: 5, DefaultColor: "#A0A0A0", Hide: ["Glasses"], Effect: ["BlindLight", "Prone"] },
 			{ Name: "LeatherBlindfold", Fetish: ["Leather"], Value: 30, Time: 5, AllowLock: true, DefaultColor: "#404040", Hide: ["Glasses"], Effect: ["BlindNormal", "Prone"] },
+			{ Name: "InteractiveVisor", Fetish: ["Metal"], BuyGroup: "FuturisticVisor", Value: 50, Time: 5, AllowLock: true, Hide: ["ItemNoseNoseRing", "MaskFuturisticVisor"], Effect: ["Prone"], AllowEffect: ["BlindNormal", "BlindHeavy", "BlindLight"], AllowType: ["LightTint", "HeavyTint", "Blind", "Face"], Extended: true,
+				Layer: [
+					{ Name: "Mask", AllowColorize: true },
```
Comment (ace-1331) [inline BondageClub/Assets/Female3DCG/Female3DCG.js:2853]: Any reason why there is a single layer? It's not necessary, the layer can be removed to simplify the code

File: BondageClub/Assets/Female3DCG/Female3DCG.js [hunk 2]
```diff
@@ -2848,6 +2848,11 @@ var AssetFemale3DCG = [
 		Asset: [
 			{ Name: "ClothBlindfold", Value: 15, Time: 5, DefaultColor: "#A0A0A0", Hide: ["Glasses"], Effect: ["BlindLight", "Prone"] },
 			{ Name: "LeatherBlindfold", Fetish: ["Leather"], Value: 30, Time: 5, AllowLock: true, DefaultColor: "#404040", Hide: ["Glasses"], Effect: ["BlindNormal", "Prone"] },
+			{ Name: "InteractiveVisor", Fetish: ["Metal"], BuyGroup: "FuturisticVisor", Value: 50, Time: 5, AllowLock: true, Hide: ["ItemNoseNoseRing", "MaskFuturisticVisor"], Effect: ["Prone"], AllowEffect: ["BlindNormal", "BlindHeavy", "BlindLight"], AllowType: ["LightTint", "HeavyTint", "Blind", "Face"], Extended: true,
```
Comment (ace-1331) [inline BondageClub/Assets/Female3DCG/Female3DCG.js:2851]: There's an extra allowtype that should be removed (`"Face"`)

**Review by ace-1331 — CHANGES_REQUESTED** [review]
Besides the code comments, the rest looks good.

**Comment by Ada18980** [comment]
Resolved both issues and re-tested.

**Review by ace-1331 — APPROVED** [review]

**Comment by Ben987** [comment]
Absolutely!  I will include it in the next beta, thanks :)

**Pull request merged**